openrtb_ext/bid_request_video.go

```diff
@@ -0,0 +1,261 @@
+package openrtb_ext
+
+import (
+	"github.com/mxmCherry/openrtb"
+)
+
+type BidRequestVideo struct {
+	// Attribute:
+	//   accountid
+	// Type:
+	//   string; required
+	// Description:
+	//   Unique ID of the stored request
+	AccountId string `json:"accountid"`
+
+	// Attribute:
+	//   storedrequestid
+	// Type:
+	//   string; required
+	// Description:
+	//   Unique ID of the stored request
+	StoredRequestId string `json:"storedrequestid"`
+
+	// Attribute:
+	//   podconfig
+	// Type:
+	//   object; required
+	// Description:
+	//   Container object for describing all the pod configurations
+	PodConfig PodConfig `json:"podconfig"`
+
+	// Attribute:
+	//   app
+	// Type:
+	//   object; App or Site required
+	// Description:
+	//   Application where the impression will be shown
+	App openrtb.App `json:"app"`
+
+	// Attribute:
+	//   site
+	// Type:
+	//   object; App or Site required
+	// Description:
+	//   Site where the impression will be shown
+	Site openrtb.Site `json:"site"`
+
+	// Attribute:
+	//   user
+	// Type:
+	//   object; optional
+	// Description:
+	//   Container object for the user of of the actual device
+	User SimplifiedUser `json:"user,omitempty"`
```

Comment:
The "omitempty" annotation should be used on pointers to structs rather than nested structs so value can be nil if not given when unmarshalling or completely omitted if nil when marshalling. Note this is the case for many fields in these structs, however "omitempty" is ok for a bool/int/string field that would default to false/zero/"" if it doesn't exist in the json.